README.md

```diff
@@ -18,8 +18,35 @@ MiniModelarDB intentionally does not gather usage data. So, all users are highly
 encouraged to post comments, suggestions, and bugs as GitHub issues, especially
 if a limitation of MiniModelarDB prevents it from being used in a particular domain.
 
+## Installation
+### Linux
+The following commands are for Ubuntu Server. However, equivalent commands should work for other Linux distributions.
+
+1. Install [build-essential](https://packages.ubuntu.com/jammy/build-essential): `sudo apt install build-essential`
+2. Install [CMake](https://cmake.org/): `sudo apt install cmake`
+
+### macOS
+1. Install the Xcode Command Line Developer Tools: `xcode-select --install`
+2. Install [CMake](https://cmake.org/) and follow the _"How to Install For Command Line Use"_ menu item.
+
+### Windows
+1. Install a supported version of [Visual Studio](https://visualstudio.microsoft.com/vs/older-downloads/) with Visual C++:
+   * Visual Studio 2017 ([Supported](https://github.com/microsoft/snmalloc/blob/main/docs/BUILDING.md#building-on-windows))
+   * Visual Studio 2019 ([Supported](https://github.com/microsoft/snmalloc/issues/525#issuecomment-1128901104))
+   * Visual Studio 2022 (Unsupported)
```

Comment:
Why is is listed as the others (under supported versions) if is not supported?

Should it rather be "Note that Visual Studio 2022 is not supported"?

Reply:
Fixed.